backend/sources/google/events-router/src/main/java/co/airy/core/sources/google/WebhookEvent.java

```diff
@@ -0,0 +1,49 @@
+package co.airy.core.sources.google;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Objects;
+import java.util.stream.Stream;
+
+@Data
+@NoArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class WebhookEvent {
+    private String agent;
+
+    @JsonProperty("conversationId")
+    private String conversationId;
+    @JsonProperty("customAgentId")
+    private String customAgentId;
+
+    @JsonProperty("context")
+    private JsonNode context;
+    @JsonProperty("message")
+    private JsonNode message;
+
+    @JsonProperty("surveyResponse")
+    private JsonNode surveyResponse;
+    @JsonProperty("suggestionResponse")
+    private JsonNode suggestionResponse;
+    private JsonNode receipts;
+    @JsonProperty("userStatus")
+    private JsonNode userStatus;
+
+    @JsonProperty("sendTime")
+    private String sendTime;
```

Comment:
(honest question) why are the JsonProperty needed here? Because of the default setup of our object mapper? I'm also wondering then why not all the properties have the annotation

Reply:
Good question. I removed the annotations and the webhook event is not parseable any more. So it's definitely needed